reflectivity_ui/interfaces/data_handling/instrument.py

```diff
@@ -424,3 +425,28 @@ def integrate_detector(self, ws, specular=True):
         integrated = api.Integration(ws_summed)
         integrated = api.Transpose(integrated)
         return integrated
+
+    @staticmethod
```

Comment:
Do we really need this decorator here? The function is used only once in the same class

Reply:
I made it staticmethod to silence my IDE that complained that it was a class method that didn't use any class variables. I now instead moved it outside the class, since it is general and could be used elsewhere.